pkg/cmd/roachtest/test_impl.go

```diff
@@ -157,6 +158,13 @@ func (t *testImpl) Cockroach() string {
 		t.l.Printf("Benchmark test, running with standard cockroach")
 		return t.StandardCockroach()
 	}
+	if t.spec.Operation {
```

Comment:
Can you clarify what the use case for this is? Operations shouldn't be querying for local cockroach paths, as they are not uploading binaries.

Reply:
The purpose would be to call `cockroach` commands on the remote node, eg. `cockroach node decommission`. For that we need to relay the passed-in cockroach binary path from roachtestflags through to this method.

Reply:
Hrmm, there is a bit of a mismatch here. For context, `t.Cockroach()` in roachtest runs translates to "path to cockroach binary _in the test runner_" (i.e., not in the cluster). This used to be needed because tests used to have to upload cockroach before starting the cluster, a repetitive step that has since been removed. We should have removed the `Cockroach()` function from the test interface.

From what I understand, your purpose is for operations to do something like `c.Run("%s workload run", t.Cockroach())`. If we were to do that, then it would 1) be confusing if you knew the previous meaning; and 2) also make it harder to call an operation from roachtest.

Any chance operations can assume a `./cockroach` path for now? My understanding is that we will want to be able to pass parameters to operations at some point, and the path to the binary on the cluster could be a parameter that we pass.